# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-04-02 06:52+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: spotify/views.py:41
msgid "spotify_client_secret_error"
msgstr "SPOTIFY_CLIENT_ID et SPOTIFY_CLIENT_SECRET doivent être fournis."

#: spotify/views.py:77 spotify/views.py:88 spotify/views.py:138
#: spotify/views.py:141
msgid "user_not_found"
msgstr "Utilisateur non trouvé."

#: spotify/views.py:83
msgid "state_not_found"
msgstr "État non trouvé."

#: spotify/views.py:103 spotify/views.py:165
msgid "access_token_error"
msgstr "Échec de la récupération du jeton d'accès."

#: spotify/views.py:133
msgid "user_id_not_found"
msgstr "Identifiant utilisateur non trouvé."

#: spotify/views.py:273
msgid "playlist_with_tracks_error"
msgstr "Erreur lors de la récupération de la liste de lecture."

#: spotify/views.py:295
msgid "recently_played_error"
msgstr "Erreur lors de la récupération de la liste de lecture."
